oppia_android_test.bzl

```diff
@@ -1,41 +1,48 @@
-load("@rules_jvm_external//:defs.bzl", "artifact")
-load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
-
-def oppia_android_test(name, srcs, test_manifest, custom_package, test_class, deps,
-                       enable_data_binding=False, assets=None, assets_dir=None):
-  '''
-  Creates an Oppia test target for running the specified test as an Android local test with Kotlin
-  support. Note that this creates an additional, internal library.
+"""
```

Comment:
Lint failure we get without this fix

```
oppia_android_test.bzl:1: module-docstring: The file has no module docstring.
A module docstring is a string literal (not a comment) which should be the first statement of a file (it may follow comment lines). (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#module-docstring)
oppia_android_test.bzl # reformat
```

Reply:
LGTM.